app/views/planning_applications/validate_documents_form.html.erb

```diff
@@ -1,32 +1,87 @@
 <% content_for :title, "Validate documents" %>
-
 <%= render "planning_applications/assessment_dashboard" do %>
 
-  <h2 class="govuk-heading-m govuk-!-padding-top-3">Validate application</h2>
+<h2 class="govuk-heading-m govuk-!-padding-top-3">Validate application</h2>
 
-  <p class="govuk-body"><%= link_to "Check documents", planning_application_documents_path(@planning_application) %></p>
+<p class="govuk-body">
+  <%= link_to "Check documents", planning_application_documents_path(@planning_application) %>
+</p>
 
-  <%= form_with model: @planning_application, url: validate_documents_planning_application_path(@planning_application), local: true, builder: GOVUKDesignSystemFormBuilder::FormBuilder do |form| %>
-    <fieldset class="govuk-fieldset">
-      <% if form.object.errors.any? %>
-        <% form.object.errors.each do |error| %>
-      <span id="status-error" class="govuk-error-message">
-        <span class="govuk-visually-hidden">Error:</span><%= error.message %></span>
-        <% end %>
+<% if @planning_application.description_change_requests.present? %>
+  <table class="govuk-table change-requests">
+    <thead class="govuk-table__head">
+      <tr class="govuk-table__row">
+        <th scope="col" class="govuk-table__header govuk-!-width-one-quarter">
+          Request
+        </th>
+        <th scope="col" class="govuk-table__header govuk-!-width-one-quarter">
+          Time remaining
+        </th>
+        <th scope="col" class="govuk-table__header govuk-!-width-one-quarter">
+          Status
+        </th>
+      </tr>
+    </thead>
+    <tbody class="govuk-table__body">
+      <% @planning_application.description_change_requests.each do |description_change_request| %>
```

Comment:
More of a product question, but do we want to order these with Open ones at the top, regardless of date? In that case, we'd need a scope

Reply:
really good point, that actually reminds me they should be grouped according to the date they were sent in per one of the designs! I'll create a scope for that and double check with ali if she wants them ordered by open or by date